addons/monitoring/lambda/main.go

```diff
@@ -55,6 +55,7 @@ type OptionsStruct struct {
 	AWSRegion                  string `long:"aws-region" env:"AWS_REGION" required:"true"`
 	CronDelayTolerance         string `long:"cron-delay-tolerance" env:"CRON_DELAY_TOLERANCE" default:"2h"`
 	CronMonitorInterval        string `long:"monitor-run-interval" env:"CRON_MONITOR_RUN_INTERVAL" default:"1 hour"`
+	AwsEndpointUrl             string `long:"aws-endpoint-url" env:"AWS_ENDPOINT_URL"`
```

Comment:
Adding this to allow easier testing of this in the future, since the Go AWS SDK doesn't support the env var natively yet like it does for e.g. `AWS_ACCESS_KEY_ID`